portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,37 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import com.google.auto.value.AutoValue;
+
+/** Creates a Comment for each comment written in portfolio. */
```

Comment:
Sorry to comment on this again, but this class does not create comments, that is what the constructor does. This class stores data related to comments written in the porfolio.

/** Stores data for comments written in portfolio. */

Reply:
done